ably/http/http.py

```diff
@@ -50,6 +50,7 @@ def __init__(self, method='GET', url='/', headers=None, body=None, skip_auth=Fal
         self.set_version_headers()
 
     def set_version_headers(self):
+        # RSC7a, RSC7b
```

Comment:
Also, these spec item codes should exist with the tests, not in the code as the code does not "prove" the spec is met, the test does.